HPXMLtoOpenStudio/resources/EPvalidator.xml

```diff
@@ -1078,7 +1078,8 @@
       <sch:assert role='ERROR' test='h:FuelType[text()="natural gas" or text()="fuel oil" or text()="fuel oil 1" or text()="fuel oil 2" or text()="fuel oil 4" or text()="fuel oil 5/6" or text()="diesel" or text()="propane" or text()="kerosene" or text()="coal" or text()="coke" or text()="bituminous coal" or text()="anthracite coal" or text()="electricity" or text()="wood" or text()="wood pellets"] or not(h:FuelType)'>Expected FuelType to be 'natural gas' or 'fuel oil' or 'fuel oil 1' or 'fuel oil 2' or 'fuel oil 4' or 'fuel oil 5/6' or 'diesel' or 'propane' or 'kerosene' or 'coal' or 'coke' or 'bituminous coal' or 'anthracite coal' or 'electricity' or 'wood' or 'wood pellets'</sch:assert>
       <sch:assert role='ERROR' test='count(h:TankVolume) &lt;= 1'>Expected 0 or 1 element(s) for xpath: TankVolume</sch:assert>
       <sch:assert role='ERROR' test='count(h:HeatingCapacity) &lt;= 1'>Expected 0 or 1 element(s) for xpath: HeatingCapacity</sch:assert>
-      <sch:assert role='ERROR' test='count(h:UniformEnergyFactor) + count(h:EnergyFactor) = 1'>Expected 1 element(s) for xpath: UniformEnergyFactor | EnergyFactor</sch:assert>
+      <sch:assert role='ERROR' test='count(h:UniformEnergyFactor) + count(h:EnergyFactor) &lt;= 1'>Expected 0 or 1 element(s) for xpath: UniformEnergyFactor | EnergyFactor</sch:assert>
+      <sch:assert role='ERROR' test='count(h:EnergyFactor) + count(h:YearInstalled) &gt;= 1 or count(h:UniformEnergyFactor) = 1'>Expected 1 or more element(s) for xpath: (EnergyFactor | YearInstalled) | UniformEnergyFactor</sch:assert>
```

Comment:
Not sure if this is right. What I want to test here is:
- UniformEnergyFactor and EnergyFactor are optional.
- YearInstalled only required if both UniformEnergyFactor and EnergyFactor are not provided.
- EnergyFactor only required if both UniformEnergyFactor and YearInstalled are not provided. 
- UniformEnergyFactor is only required if both YearInstalled and EnergyFactor are not provided.